### PR TITLE
last changes from old repository

### PR DESCRIPTION
**Pull request by radsoulbeard**

### CODE REVIEW

**Comment by 1084565** [comment]
did you cross check with #32 ?

**Comment by radsoulbeard** [comment]
> did you cross check with #32 ? I put the documentation of the docker file into the master branch. Again, I think its unfortunate to have the Dockerfile hidden in another branch, so at least the doc should be in `master`.

There should also be a README.md in the docker branch, as this is taken as a description for dockerhub.

**Comment by 1084565** [comment]
Good point. I'll also check #32 to make make it mergeable if necessary.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+This will print the CM Client's help information. For a comprehensive overview of available commands please read the [documentation](https://github.com/SAP/devops-cm-client#usage).
+
+# How to build this image
+
+`docker build -t cm-client .`
+
+## This image provides:
+
+- [CM CLient](https://github.com/SAP/devops-cm-client)
```
Comment (1084565) [inline README.md]:
```suggestion
- [CM Client](https://github.com/SAP/devops-cm-client)
```

**Pull request merged**